resource_booking/models/resource_booking.py

```diff
@@ -125,7 +125,12 @@ class ResourceBooking(models.Model):
     description = fields.Html()
     partner_id = fields.Many2one(
         comodel_name="res.partner",
+        compute="_compute_partner_id",
         string="Requester",
+    )
+    partner_ids = fields.Many2many(
+        comodel_name="res.partner",
+        string="Requesters",
         index=True,
```

Comment:
`index` doesn't make sense in m2m, not `ondelete`.

Reply:
so, should i keep ondelete?

Reply:
Please remove it